tasks/README.md

```diff
@@ -104,8 +107,13 @@ If a parameter type accepts `null` the task runner will accept either a `null` v
 
 ### Metaparameters
 
-In addition to the tasks parameters, the task runner may inject metaparameters prefixed by '_'. These include `_noop` and `_task`.
+In addition to the tasks parameters, the task runner may inject metaparameters prefixed by `_`.
+
+**_noop**: Used to implement logic in tasks to support cases where task should not perform certain actions.
 
+**_task**: Allow multiple task implementations to access the same executable file. The `_task` metaparameter provides the executable the task name to allow task specific logic to be implemented. 
+
+**_installdir**: Tasks with `files` specified in the metatdata will be passed the `_installdir` metaparam to provide the file path to the expected executables. 
```

Comment:
I'd like to see an example of using that, to clarify what full path including `_installdir` would look like for some example paths. Do files get flattened? Do they preserve some directory structure? What does it look like?